### ISSUE TITLE
[Programmatic Access - Make code for micro bit - My Projects]: User-scalable=no is defined for "viewport" under <meta> tag.

### ISSUE THREAD

**Issue by Chandangithb** [microsoft/pxt-microbit#4964]
### User Experience:
People with visual disabilities, low vision, or limited color perception are likely to find text unreadable when it’s too small. As most people find it easier to read text when it is sufficiently large.

**Note:** User credentials should NOT be included in the bug.

### Repro Steps:

1. Open https://makecode.microbit.org/
2. Navigate to header controls such as " micro: bit, Dropdown menu More..." etc.
3. Run AI tool.
4. Observe and verify the issue.

### Actual Result:

**Issue:** User-scalable=no is defined for "viewport" under <meta> tag.

### Expected Result:
**Solution:** User-scalable=no should be removed present under <meta> tag.

### Issue:

Ensures <meta name="viewport"> does not disable text scaling and zooming (meta-viewport - https://accessibilityinsights.io/info-examples/web/meta-viewport)

### Target application:

Microsoft MakeCode for micro:bit - https://makecode.microbit.org/

### Element path: 

meta[name="viewport"]

### Snippet: 
`<meta name="viewport" content="width=device-width,height=device-height,user-scalable=no,initial-scale=1.0,maximum-scale=1.0,minimum-scale=1.0">`

### How to fix: 

**Fix any of the following:**
  user-scalable=no on <meta> tag disables zooming on mobile devices

### Environment:
Microsoft Edge version 111.0.1660.14

====

This accessibility issue was found using Accessibility Insights for Web 2.37.3 (axe-core 4.6.3), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

### MAS Reference:
[MAS 1.4.4 – Resize Text](https://aka.ms/MAS1.4.4)

### Reference Links:

- [Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
- External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [External Bug Process (sharepoint.com)](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx?xsdata=MDN8MDF8fDE1MThmM2JlZGFjMDRlZDhiZmViYTUxMDA2M2NiZWI3fDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2Mzc3NDEyNTM2MTEzNzIwNzR8R29vZHxWR1ZoYlhOVFpXTjFjbWwwZVZObGNuWnBZMlY4ZXlKV0lqb2lNQzR3TGpBd01EQWlMQ0pRSWpvaUlpd2lRVTRpT2lJaUxDSlhWQ0k2TVRKOQ%3D%3D&sdata=MnlpSXlVMnpGT3cvZ0xJNnFWUWZWTGt6dmo1VkJMZFpIZHBvUUduNDhyMD0%3D&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2cv-kgeetala%40microsoft.com). “HCL Staff should not log any third-party external bugs. Should be routed to EDAD team”.
- Please reach out to [C&AI Teams channel](https://apc01.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cAklavyaM%40hcl.com%7c24e89c6ac26f42cfe40c08d7667e49f3%7c189de737c93a4f5a8b686f4ca9941912%7c0%7c0%7c637090564906844600&sdata=22JjBMoA1ID/KnYqFa/OB%2BrkTaLh5YYsGBd/7Bm7u5w%3D&reserved=0) for any process related queries.

### Test Environment:

- OS Version: 22H2 (OS Build:22621.1265)
- Browser: Microsoft Edge Version 111.0.1660.14 (Official build) (64-bit)
- Tool used: Accessibility insights for Web, Developer Tool (F12)
- #URL: https://makecode.microbit.org/
- 
![Autoamated checks result(meta tag)](https://user-images.githubusercontent.com/112375678/221908391-ce6cafd5-9404-4efa-9493-86ff02ebb13d.jpg)
![MAS1 4 4-User-scalable=no is defined for 'viewport' under meta tag](https://user-images.githubusercontent.com/112375678/221908401-eed257a3-9902-43dd-a47f-bc69ca039c70.jpg)

**Comment by Chandangithb** [comment]
#A11yAuto, #A11yMAS, #A11ySev3, #HCL, #Accessibility, #MAS1.4.4, #K4W, #HCL-MakeCode-New, #WCP

**Comment by testingmakecode** [comment]
#A11yAuto;#A11yMAS;#A11ySev2;#HCL;#Accessibility;#MAS1.4.4;#K4W;#HCL-MakeCode-New;#WCP

**Comment by testingmakecode** [comment]
As checked on [Microsoft Make Code Arcade] (https://arcade.makecode.com/) the issue is still repro. #Regressed:09--28-23 #A11ySev3

**Comment by srietkerk** [comment]
Mobile is not supported.
